### PR TITLE
fixed typos for modfinder chapter

### PR DESCRIPTION
**Pull request by jjgao**

### CODE REVIEW

**Comment by jjgao** [comment]
@lafita @pwrose : would you please review the merge the pull request? Thanks.

File: modfinder/add-protein-modification.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-How to define a new protein modiifcation?
+How to define a new protein modifcation?
```
Comment (andreasprlic) [inline modfinder/add-protein-modification.md]: typo: should be modification

File: modfinder/add-protein-modification.md [hunk 2]
```diff
@@ -84,7 +84,7 @@ ProteinModificationXmlReader.registerProteinModificationFromXml(fis);
 Navigation:
 [Home](../README.md)
 | [Book 6: The ModFinder Modules](README.md)
-| Chapter 4 - How to define a new protein modiifcation
+| Chapter 4 - How to define a new protein modifcation
```
Comment (andreasprlic) [inline modfinder/add-protein-modification.md]: same typo again

File: modfinder/identify-protein-modifications.md [hunk 3]
```diff
@@ -72,4 +72,4 @@ Navigation:
 
 Prev: [Chapter 2 : How to get a list of supported protein modifications](supported-protein-modifications.md)
 
-Next: [Chapter 4 : How to define a new protein modiifcation](add-protein-modification.md)
+Next: [Chapter 4 : How to define a new protein modifcation](add-protein-modification.md)
```
Comment (lafita) [inline modfinder/identify-protein-modifications.md]: same typo that @andreasprlic described.

**Pull request merged**